source/changelog/r85.inc

```diff
@@ -0,0 +1,42 @@
+.. _changelog-r85:
+
+*************************
+Release Feburary 9th 2021
+*************************
+The following changes started rolling out on Feburary 9th. All backend/Api changes have been deployed while Client changes are propagating gradually over the next couple of days.
+
+
+API and Backend
+===============
+The following new features and improvements are now available in the API and the backend.
+
+* **Breaking** Added additional order and plan validation to prevent orders for products with no price line. This means products that are free of charge now need to register a price of 0. All existing products in production already have price 0 but new ones will break if not configured correctly.
```

Comment:
I think the only way to break this in the client is to delete the price, and I'm not sure that is possible either. 
So I think it's more a problem for 3rd party users and for our internal testing.